18F/Head-Start-TTA
# Remove Elasticsearch dependency

**Description of change**

Remove dependency on Elasticsearch. We will run all AR searches through Postgres now.

**How to test**

1) View grantee details page
2) Enter query
3) Ensure results match what you expect

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/0

**Checklist**

    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
